docs.ps1

```diff
@@ -0,0 +1,53 @@
+param([Switch]$D = $false)
```

Comment:
```suggestion
param(
    [Switch]$BuildDrafts
)
```

In PowerShell, it's good practice to use longer, descriptive parameters where possible. For `switch` parameters in particular, no default is needed as they are false unless specified.